cmd/version.go

```diff
@@ -14,10 +14,9 @@
 package cmd
 
 import (
-	"fmt"
-
 	"github.com/okteto/okteto/cmd/utils"
 	"github.com/okteto/okteto/pkg/config"
+	"github.com/okteto/okteto/pkg/log"
```

Comment:
Preferably this would be `oktetoLog` as there's a built-in `log` with a well-known api.

Reply:
fixed at https://github.com/okteto/okteto/pull/2090/commits/d6992e7ceb1e7a4ce52b30c04186665b1bc135b7